src/main/java/com/redhat/management/approval/EmailBody.java

```diff
@@ -170,6 +176,26 @@ private static String getUrlContent(URL url) throws Exception {
         return content.toString();
     }
 
+    private String encrypt(String strToEncrypt) throws Exception {
+        String v2Key = System.getenv("ENCRYPTION_KEY");
```

Comment:
@bzwei do we need to add a task to add this into
https://console.insights-dev.openshift.com/console/project/approval-ci/browse/dc/approval-pam?tab=environment

Reply:
https://github.com/RedHatInsights/e2e-deploy/pull/1145